README.md

```diff
@@ -6,6 +6,10 @@ Browse and query your MongoDB databases both locally and in the cloud using [_sc
 
 ![Cosmos DB Extension](resources/features.png)
 
+# We've Moved!
+
+You will now find us by clicking in the new Azure view container instead of the file explorer.
+

```

Comment:
Is there an image here we need to display?

Reply:
Matt didn't want it.